src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,44 @@
 package core.basesyntax;
 
-class Pair {
+public class Pair<T, S> {
+    private T item1;
+    private S item2;
```

Comment:
let's not use numbers in variable naming

Reply:
solved